locales-src/extra_aliases.js

```diff
@@ -158,6 +158,15 @@ export default {
     अंत: "scratchblocks:end",
   },
 
+  hu: {
+    // Hungarian
+    "fordulj balra %1 fokot": "MOTION_TURNLEFT",
+    "fordulj jobbra %1 fokot": "MOTION_TURNRIGHT",
+    "zászlóra kattintáskor": "EVENT_WHENFLAGCLICKED",
+    "a zászlóra kattintáskor": "EVENT_WHENFLAGCLICKED",
```

Comment:
I don't speak Hungarian, but this looks plausible to me!

<img width="153" alt="Screenshot 2023-03-13 at 10 22 42" src="https://user-images.githubusercontent.com/639289/224674497-530dd931-4524-4027-820c-815d562a6e88.png">